config/ompi_check_libhcoll.m4

```diff
@@ -59,7 +52,15 @@ AC_DEFUN([OMPI_CHECK_HCOLL],[
 			   CPPFLAGS=$CPPFLAGS_save
 			   LDFLAGS=$LDFLAGS_save
 			   LIBS=$LIBS_save],
-			   [ompi_check_hcoll_happy="no"])
+			  [OPAL_CHECK_PACKAGE([$1],
```

Comment:
The same kind of comments that I made in the fca configure.m4 (above) also apply here.